packages/apputils/src/iframe.ts

```diff
@@ -10,9 +10,55 @@ export class IFrame extends Widget {
   /**
    * Create a new IFrame widget.
    */
-  constructor() {
+  constructor(options: IFrame.IOptions = {}) {
     super({ node: Private.createNode() });
     this.addClass('jp-IFrame');
+    this._exceptions = options.exceptions || [];
+    this.sandbox = !!options.sandbox;
+  }
+
+  /**
+   * Whether the iframe is sandboxed.
+   */
+  get sandbox(): boolean {
+    const iframe = this.node.querySelector('iframe')!;
+    const attr = iframe.getAttribute('sandbox');
+    return attr !== null && attr !== undefined; // An empty string is sandboxed.
+  }
+  set sandbox(value: boolean) {
```

Comment:
What about collapsing these to just `sandbox` as a list of types, with "secure as we can make it" by default?

Reply:
Sure, that makes sense to me. More security by default.